ocs_ci/helpers/helpers.py

```diff
@@ -3334,3 +3334,22 @@ def get_failure_domain():
 
     storage_cluster_obj = get_storage_cluster()
     return storage_cluster_obj.data["items"][0]["status"]["failureDomain"]
+
+
+def get_pod_logs(
```

Comment:
There is already a function "get_pod_logs" in ocs-ci/ocs_ci/ocs/resources/pod.py:1107
Please consider using in case needed

Reply:
done